### PR TITLE
fix: code block backspace error and task list error

### PR DESCRIPTION
**Pull request by Jocs**
| Q                | A
| ---------------- | ---
| Bug fix?         | yes
| Fixed tickets    | #918  #738 
| License          | MIT

### Description

Pls see the issue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "build:dir": "node .electron-vue/build.js && electron-builder --dir",
     "build:clean": "cross-env BUILD_TARGET=clean node .electron-vue/build.js",
     "build:dev": "node .electron-vue/build.js",
-    "dev": "node .electron-vue/dev-runner.js",
+    "dev": "cross-env MARKTEXT_ERROR_INTERACTION=0 node .electron-vue/dev-runner.js",
```
Comment (fxha) [inline package.json]: I prefer that you set `MARKTEXT_ERROR_INTERACTION` in your environment variables.

Comment (Jocs) [reply package.json]: Ok!

**Comment by Jocs** [comment]
@fxha `ctrl + A` is an evil...So many issues are relate to it...

**Comment by Jocs** [comment]
>I cannot reproduce the code block language identifier issue but I found another issue produced by this PR:
>Write - [ ] a
>Ctrl+A (result in - [ ] )
>Ctrl+A again (result: task list item still exists but is empty) - bug 1
>If you press backspace 1 or 2 times again --> editor broken - bug 2

Updated. pls review.

**Pull request merged**